### PR TITLE
Installing enzyme

### PR DESCRIPTION
**Pull request by NathanCorbin**
issue #14 

**Description**
Query blog posts using graphql

**Checklist (check all that is appropriate)**

[X] Issue is linked in this PR
[X] Existing tests pass
[ ] Accessibility tested (if appropriate)

### CODE REVIEW

File: src/components/blog-highllights/exerpt-list/ExerptList.tsx [hunk 1]
```diff
@@ -1,61 +1,20 @@
 import * as React from 'react'
 import { CardBody, CardTitle, CardDesctiption, CardMeta } from '../../card'
 
-export const ExerptList: React.FC = () => {
+const ExerptList: React.FC<any> = props => {
```
Comment (QuentinGuenther) [inline src/components/blog-highllights/exerpt-list/ExerptList.tsx:4]: Could you add some interfaces for the <any> type here?

File: src/components/blog-highllights/exerpt-list/ExerptList.tsx [hunk 2]
```diff
@@ -1,61 +1,20 @@
...
-        />
-        <CardDesctiption
-          text="Lorem ipsum dolor sit amet, consectetur adipiscing elit ut aliquam, purus sit amet luctus venenatis,
-                            lectus magna fringilla urna, porttitor rhoncus dolor purus non enim praesent elementum facilisis leo,
-                            vel fringilla est ullamcorper eget nulla facilisi etiam dignissim diam quis enim lobortis scelerisque
-                            fermentum dui faucibus in ornare quam viverra"
-        />
-        <CardMeta publishDate="November 11, 2016" />
-      </CardBody>
+      {allMarkdownRemark.allMarkdownRemark.edges.map((post: any) => (
```
Comment (QuentinGuenther) [inline src/components/blog-highllights/exerpt-list/ExerptList.tsx:8]: Could you add some interfaces for the <any> type here?

File: src/components/blog-highllights/exerpt-list/ExerptList.tsx [hunk 3]
```diff
@@ -1,61 +1,20 @@
...
+        <CardBody>
+          <CardTitle title={post.node.frontmatter.title} backgroundUrl={post.node.frontmatter.banner} />
+          <CardDesctiption text={post.node.frontmatter.description} />
+          <div dangerouslySetInnerHTML={{ __html: post.node.html }} />
```
Comment (QuentinGuenther) [inline src/components/blog-highllights/exerpt-list/ExerptList.tsx:12]: We don't need to include the blog post content(html) in this component

File: src/components/blog-highllights/exerpt-list/__tests__/ExerptList.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import ExerptList from '../ExerptList'
+
+it('renders correctly', () => {
+  const query = {
```
Comment (QuentinGuenther) [inline src/components/blog-highllights/exerpt-list/__tests__/ExerptList.spec.tsx:6]: Could you move this to `src/components/blog-highllights/exerpt-list/__tests__/mocks/card-data.ts`

File: tsconfig.json [hunk 5]
```diff
@@ -12,7 +12,8 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "removeComments": false,
-    "preserveConstEnums": true
+    "preserveConstEnums": true,
+    "allowSyntheticDefaultImports": true
```
Comment (QuentinGuenther) [inline tsconfig.json:16]: What is this for?

**Comment by QuentinGuenther** [comment]
Could you add the purpose of these changes in the PR description, issue#, and all that?

File: src/components/blog-highllights/exerpt-list/ExerptList.tsx (hunk 1 above)
Comment (NathanCorbin) [reply src/components/blog-highllights/exerpt-list/ExerptList.tsx:4]: Yes

File: src/components/blog-highllights/exerpt-list/ExerptList.tsx (hunk 2 above)
Comment (NathanCorbin) [reply src/components/blog-highllights/exerpt-list/ExerptList.tsx:8]: No, gives an error about props destructuring

File: src/components/blog-highllights/exerpt-list/ExerptList.tsx (hunk 3 above)
Comment (NathanCorbin) [reply src/components/blog-highllights/exerpt-list/ExerptList.tsx:12]: Will remove

File: src/components/blog-highllights/exerpt-list/__tests__/ExerptList.spec.tsx (hunk 4 above)
Comment (NathanCorbin) [reply src/components/blog-highllights/exerpt-list/__tests__/ExerptList.spec.tsx:6]: `jest` thinks `card-data.ts` is part of the test suite and fails. I had to move the `__mocks__` folder outside the `__tests__ ` folder for the tests to pass.

File: tsconfig.json (hunk 5 above)
Comment (NathanCorbin) [reply tsconfig.json:16]: No clue.

Comment (QuentinGuenther) [reply tsconfig.json]: Fair enough

File: src/components/blog-highllights/exerpt-list/ExerptList.tsx (hunk 2 above)
Comment (QuentinGuenther) [reply src/components/blog-highllights/exerpt-list/ExerptList.tsx]: Fixed

**Pull request merged**